proxies/nodejs/app.ts

```diff
@@ -1,49 +1,53 @@
-import { DVCClient, DVCUser } from '@devcycle/nodejs-server-sdk'
-import Koa from 'koa'
-import Router from 'koa-router'
-import bodyParser from 'koa-bodyparser'
-import { handleUser } from './handlers/user'
-import { handleClient } from './handlers/client'
+import { DVCClient, DVCUser } from "@devcycle/nodejs-server-sdk";
```

Comment:
whats up with the linting in this PR? It looks like it should be single quotes and no semi-colons from what I can tell from the closed linting PR